app/routes/activity/components/components/Invoice/Invoice.js

```diff
@@ -13,16 +13,14 @@ const Invoice = ({ invoice, ticker, currentTicker, showActivityModal, currencyNa
     className={`${styles.container} ${!invoice.settled && styles.unpaid}`}
     onClick={() => showActivityModal('INVOICE', { invoice })}
   >
-    {!invoice.settled && (
-      <div className={styles.activityTypeIcon}>
-        <section
-          className="hint--bottom"
-          data-hint={`Lightning invoice${!invoice.settled ? ' (unpaid)' : undefined}`}
-        >
-          <Isvg src={checkmarkIcon} />
-        </section>
-      </div>
-    )}
+    <div className={styles.activityTypeIcon}>
+      <section
+        className="hint--bottom"
+        data-hint={`Lightning invoice${!invoice.settled ? ' (unpaid)' : undefined}`}
```

Comment:
Are you not using `!invoice.settled && ' (unpaid)'` because we will soon be updating to React 16.x?

Reply:
`Lightning invoice${!invoice.settled && ' (unpaid)'}` would resolve to the following strings:

`Lightning invoice (unpaid)` for unsettled invoices
`Lightning invoicefalse` for settled invoices

Infact, with the current code we already have a bug as we have:

`Lightning invoice (unpaid)` for unsettled invoices
`Lightning invoiceundefined` for settled invoices

I will push up a fix to this PR

Reply:
Have update the PR with a fix for that.